src/ts/components/subtitlesettings/backgroundopacityselectbox.ts

```diff
@@ -18,12 +19,12 @@ export class BackgroundOpacitySelectBox extends SubtitleSettingSelectBox {
   configure(player: PlayerAPI, uimanager: UIInstanceManager): void {
     super.configure(player, uimanager);
 
-    this.addItem(null, 'default');
-    this.addItem('100', '100%');
-    this.addItem('75', '75%');
-    this.addItem('50', '50%');
-    this.addItem('25', '25%');
-    this.addItem('0', '0%');
+    this.addItem(null, i18n.t('default'));
+    this.addItem('100', i18n.t('percent', { value: 100}));
```

Comment:
Spaces within `{ … }` (All occurences)
```suggestion
    this.addItem('100', i18n.t('percent', { value: 100 }));
```